package.json

```diff
@@ -20,7 +20,8 @@
     ".": {
       "import": "./dist/vue-search-input.es.js",
       "require": "./dist/vue-search-input.umd.js"
-    }
+    },
+    "./dist/styles.css": "./dist/styles.css"
```

Comment:
Based on this issue [here](https://github.com/vitejs/vite/discussions/2657) I think the right way to add the css to exports will be like
```
    "./dist/styles.css": {
      "import": "./dist/styles.css",
      "require": "./dist/styles.css"
    }
```

Reply:
> Based on this issue [here](https://github.com/vitejs/vite/discussions/2657) I think the right way to add the css to exports will be like
> 
> ```
>     "./dist/styles.css": {
>       "import": "./dist/styles.css",
>       "require": "./dist/styles.css"
>     }
> ```

This does break it for me, but the first solution works (Vue 3.x)
    "./dist/styles.css": "./dist/styles.css"